client/src/images/flood.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 700 700" style="enable-background:new 0 0 700 700;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#0E2437;}
+	.st1{fill:#00D07B;}
+</style>
+<g transform="translate(0.000000,700.000000) scale(0.100000,-0.100000)">
+	<g>
+		<circle class="st0" cx="3490" cy="3510" r="3482.8"/>
+		<g>
+			<path class="st1" d="M5846,5502.7l-394.7-998.4H4205.7l-114.7-609.1h1208.7L4905,2902.2l-984.7-2.7l-326.4-1604.8l-1943.5,131.1
+				l1991.3,131.1L1452.2,1688l2237.2,132.5L594.5,1953l3148.2,131.1L1190,2216.6l2601.8,131.1l-3017,131.1l3068.9,132.5
+				l-2667.4,131.1l2715.2,131.1L234,3006l3706.8,131.1l-2745.2,131.1l2797.1,131.1l-2189.4,131.1l2237.2,131.1L721.5,3792.7
+				l3366.7,101.1L2103.7,4055l2036.4,131.1L560.4,4317.2L4187.9,4507l-2371,81.9l2422.9,131.1l-3063.5,131.1l3111.3,131.1
+				L1597,5113.4l2742.5,131.1l-2188,132.5l1490.1,84.7C4314.9,5500,5623.4,5513.6,5846,5502.7z"/>
+		</g>
+	</g>
+</g>
+</svg>
```

Comment:
Should minify using [`svgo`](https://github.com/svg/svgo)

Reply:
I cleaned it with `svgo --pretty flood.svg`